NewForce-at-Mountwest/bangazon-workforce-team-dog
# Computer functions

# Description

This update allows the user to see a list of Computers and all of their information, click on any feature of the computer to see that computers details.  You can create a new computer by clicking on the new computer link. Also from the details view you can delete a computer if it has never been assigned to an employee.


## Related Ticket(s)
8

## Steps to Test
-1) First, the User needs to git checkout into the jj-deleteComputer branch, then "start BangazonWorkforce" in the command line to pull up the application.
-2) Second, the User needs to find Bangazon-Workforce.sql in the Solution Explorer. Then run a new sql query to assign multiple training sessions to an employee (Dwayne).

INSERT INTO EmployeeTraining (EmployeeId, TrainingProgramId) VALUES (3,2)
INSERT INTO EmployeeTraining (EmployeeId, TrainingProgramId) VALUES (3,3)

Then click run to seed the data needed to run the application.
-3) Third, the User needs to run the application by clicking the IIS Express button at the top. This will pull up the application in the Web Browser.
-4) Next, The User should click the Departments tab in the NavBar to access the list of Computers.
At the top you can create a computer by clicking the new computer link and filling out the form. After you click submit you should see your new computer in the list.
-5) From the list, you can then click on any computer to see its details.  From the detail view you can also click delete. If you try to delete computer 2 or 3 it will give you an error message, "unable to delete this computer", if you click on the computer you just deleted, you can click delete. When the list is returned you will no longer see the computer you just deleted.

# Checklist:
- [x ] I have commented my code, particularly in hard-to-understand areas
- [x ] My changes generate no new errors
